tests/sentry/feedback/test_feedback_ingest.py

```diff
@@ -0,0 +1,191 @@
+            "sdk": {"name": "sentry.javascript.react", "version": "7.66.0-alpha.0"},
+            "user": {
+                "ip_address": "72.164.175.154",
+                "email": "josh.ferge@sentry.io",
+                "id": 880461,
+                "isStaff": False,
+                "name": "Josh Ferge",
+            },
+            "request": {
+                "url": "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d",
+                "headers": {
+                    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"
+                },
+            },
+        }
+
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(
+                path, data=missing_event_id_test_data, **self.dsn_auth_headers
+            )
+            assert response.status_code == 400
+            assert response.data == {
+                "event_id": [ErrorDetail(string="This field is required.", code="required")]
+            }
+
+    def test_wrong_type(self):
+        # Fields should be correct type
+        wrong_type_test_data = {
+            "message": 24,
+            "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+            "transaction": {},
+        }
+
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=wrong_type_test_data, **self.dsn_auth_headers)
+            assert response.status_code == 400
+            assert response.data == {
+                "transaction": [ErrorDetail(string="Not a valid string.", code="invalid")]
+            }
+
+    def test_bad_slug_path(self):
+        # Bad slug in path should lead to unsuccessful save
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path + "bad_slug", data=test_data, **self.dsn_auth_headers)
+            assert response.status_code == 404
+
+    def test_missing_optional_fields(self):
+        # Optional fields missing should still result in successful save
+        test_data_missing_optional_fields = {
+            "timestamp": 1694039635.9195,
+            "message": "This website is great!",
+            "transaction": "/replays/",
+            "type": "transaction",
+            "transaction_info": {"source": "route"},
+            "platform": "javascript",
+            "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+            "request": {
+                "url": "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d",
+                "headers": {
+                    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"
+                },
+            },
+        }
```

Comment:
I've updated the blueprint to explicitly mark what is required and what is not.

https://github.com/getsentry/sentry/pull/55566/files#diff-295e6a97402f956ed7496f0361423f12846247679848a189ab67cff310a26fb5R149